app/services/quarterly_message_employee_matcher.rb

```diff
@@ -0,0 +1,52 @@
+require "business_time"
+
+class QuarterlyMessageEmployeeMatcher
```

Comment:
nice! I love the consistency of this class with our other matcher class...wondering if we should rename the other one now to clarify difference (eg: `OnboadingMessageMatcher`